src/components/Modal/ManageFundsModal.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import ModalModel from '../../models/modalModel';
+import ModalHeader from './ModalHeader/ModalHeader';
+import {
+  GotButton,
+  ImgLink,
+  MarginedModalBody,
+  ModalWrapper,
+  Wrapper
+} from './styles';
+
+const ManageFundsModal: React.SFC<{modal: ModalModel}> = ({modal}) => {
+  return (
+    <ModalWrapper>
+      <ModalHeader onClick={modal.close} />
```

Comment:
rally? i mean `onClick={modal.close}`

Reply:
Could you clarify what's wrong with that one?